src/content/components/ProfileCallTreeSettings.js

```diff
@@ -27,17 +27,21 @@ class ProfileCallTreeSettings extends Component {
   }
 
   render() {
-    const { jsOnly, invertCallstack, searchString } = this.props;
+    const { implementationFilter, invertCallstack, searchString } = this.props;
     return (
       <div className='profileCallTreeSettings'>
         <ul className='profileCallTreeSettingsList'>
           <li className='profileCallTreeSettingsListItem'>
             <label className='profileCallTreeSettingsLabel'>
-              <input type='checkbox'
+              Filter:
+              <select
```

Comment:
actually it's not in normalize.css but in codepen's normalize.css, not sure where this comes from :)